types/geoip-lite/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for GeoIP-lite 1.1.6
+// Type definitions for GeoIP-lite 1.4.2
 // Project: https://github.com/bluesmoon/node-geoip
 // Definitions by: Yuce Tekol <http://yuce.me/>
```

Comment:
@witem add yourself here to be counted as a package owner in future PRs

Reply:
thanks, I added